lessons/beginners/install-editor/vscode.md

```diff
@@ -0,0 +1,27 @@
+{% set editor_name = 'VSCODE' %} {% set editor_url = 'https://code.visualstudio.com' %} 
+{% extends lesson.slug + '/_base.md' %}
+
+{% block name_gen %} Instalace VSCode{% endblock %}
+{% block setup %}
+
+Editor VSCode or Microsoftu podobně jako Atom funguje ihned po instalaci bez těžkého nastavování
+Pokud ale chceš v editoru psát Python kód (nebo obecně každý jazyk) nainstaluj si rozšíření pro daný jazyk. 
+
+## Jak na to:
+
+1. Otevři si VSCode editor.
+2. V sloupci úpně na levo(šedý) klikni na ikonku rozšíření. Ve výchozí pozici
+třetí od shora. Vypadá jako čtverec, který má v sobe ještě jeden.
+3. Do vyhledávacího boxu zadej "Python".
+4. Zvol první výsledek (v rohu je označen hvězdičkou).
+5. V nově otevřeném okně klikni na modré tlačítko "Install".
+6. Vyčkej, než se rozšíření nainstaluje. Editor se restartuje a zobrazí se ti okno nápovědy rozšíření.
+
+## Spuštění kódu
+
+Kód, který jsi vytvořil můžeš spustit přes zabudovaný terminál. Co to je a jak ho používat se dozvíš v dalších lekcích.
```

Comment:
Can we remove the entire section, now that it doesn't have any instructions?

Reply:
Like the entire one? Why?

Reply:
Yes, the whole section:
```
## Spuštění kódu
Kód, který jsi vytvořil můžeš spustit přes zabudovaný terminál. Co to je a jak ho používat se dozvíš v dalších lekcích.
```

Note that this will be read by people who have no idea what code is and that you can run it.